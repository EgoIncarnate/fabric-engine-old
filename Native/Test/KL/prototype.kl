/*
 *  Copyright 2010-2012 Fabric Technologies Inc. All rights reserved.
 */

function Scalar externalSquare( Scalar value ) = "externalSquare";

operator entry()
{
  report externalSquare( 7 );
}
