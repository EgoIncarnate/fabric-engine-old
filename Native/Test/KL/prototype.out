GlobalList
  Function( 'externalSquare' ('externalSquare'), Scalar:r-value )
    ParamList
      Param( 'value', Scalar, r-value )
  Function( 'entry' ('entry'), void )
    ParamList
    CompoundStatement
      StatementList
        Report
          Call('externalSquare')
            ArgList
              ConstSize( 7 )
49
